### PR TITLE
Update security.txt expires field

### LINKED ISSUE #34480: [Bug]: Update security.txt Expires field
The `Expires` field from `security.txt` should be updated regularly to make sure information is not stale.

> It is RECOMMENDED that the value of this field be less than a year into the future to avoid staleness.

https://datatracker.ietf.org/doc/html/rfc9116#section-2.5.5

Is there a way of automating this to have an up-to-date value at each major release? Or at least add the update to the release steps?

https://github.com/nextcloud/server/blob/afdb75d98b19f9d93723866dd09806e3ec1af9b6/apps/settings/lib/WellKnown/SecurityTxtHandler.php#L41

We could also add the following lines:
```
Preferred-Languages: en
Hiring: https://nextcloud.com/jobs/
```

### PR DESCRIPTION
**Pull request by nickvergessen**
Fix #34480

### CODE REVIEW

**Review by szaimen — APPROVED** [review]

**Review by tcitworld — APPROVED** [review]

File: apps/settings/lib/WellKnown/SecurityTxtHandler.php [hunk 1]
```diff
@@ -38,10 +38,11 @@ public function handle(string $service, IRequestContext $context, ?IResponse $pr
 		}
 
 		$response = "Contact: https://hackerone.com/nextcloud
-Expires: 2021-12-31T23:00:00.000Z
+Expires: 2023-04-31T23:00:00.000Z
```
Comment (BartG95) [inline apps/settings/lib/WellKnown/SecurityTxtHandler.php:41]: April has thirty days, not thirty-one....

This makes various securiy.txt checkers, such as internet.nl, tell me that the expires field is invalid.

Comment (nickvergessen) [reply apps/settings/lib/WellKnown/SecurityTxtHandler.php:41]: Ouch, that's right. Unluckily the next maintenance releases just went out, so a fox will have to wait until in 4 weeks

Comment (nickvergessen) [reply apps/settings/lib/WellKnown/SecurityTxtHandler.php:41]: Fix in https://github.com/nextcloud/server/pull/34963

**Pull request merged**